sync/awsdatabricks.py

```diff
@@ -525,6 +590,7 @@ def wait_for_final_run_status(run_id: str) -> Response[str]:
 
         sleep(30)
         run = get_default_client().get_run(run_id)
+        # TODO - log status periodically?
```

Comment:
This comment is me wondering if it makes sense to periodically log the state of the Databricks Job while we're waiting. This doesn't really add anything beyond giving a user of the CLI some confirmation that things are still progressing.